lib/dynflow/executors/parallel/core.rb

```diff
@@ -43,13 +43,13 @@ def handle_persistence_error(error)
 
         def start_termination(*args)
           super
-          logger.info 'shutting down Core ...'
+          logger.info 'shutting down Dynflow Core ...'
           @pool.tell([:start_termination, Concurrent.future])
         end
 
         def finish_termination
           @director.terminate
-          logger.error '... core terminated.'
+          logger.error '... Dynflow Core has terminated.'
```

Comment:
Good point, maybe even 'terminate' is not the right word. What about 'Dynflow Core has shut down' or something more positive :)